ansible/setup-management-network.sh

```diff
@@ -1,35 +1,53 @@
 #!/bin/bash
-
 if [[ $(id -u) -ne 0 ]]; then
     echo "Root privilege required"
     exit
 fi
 
-echo "STEP 1: Checking for bridge-utils package..."
+echo "Refreshing apt package lists..."
+apt-get update
+echo
+
+echo "STEP 1: Checking for j2cli package..."
+if ! command -v j2; then
+    echo "j2cli not found, installing j2cli"
+    apt-get install -y j2cli
+fi
+echo
+
+echo "STEP 2: Checking for bridge-utils package..."
 if ! command -v brctl; then
     echo "brctl not found, installing bridge-utils"
     apt-get install -y bridge-utils
 fi
 echo
 
-echo "STEP 2: Checking for net-tools package..."
+echo "STEP 3: Checking for net-tools package..."
 if ! command -v ifconfig; then
     echo "ifconfig not found, install net-tools"
     apt-get install -y net-tools
 fi
 echo
 
-echo "STEP 3: Checking if bridge br1 already exists..."
-if brctl show br1; then
-    echo "br1 already exists, deleting first"
-    ifconfig br1 down
-    brctl delbr br1
+echo "STEP 4: Checking for ethtool package..."
+if ! command -v ethtool; then
+    echo "ethtool not found, install ethtool"
+    apt-get install -y ethtool
+fi
+echo
+
+echo "STEP 5: Checking if bridge br1 already exists..."
+if ! brctl show br1; then
```

Comment:
If it returns the output as below, then this step fails. May be we should just add br1 even if it exists.. 

```
STEP 5: Checking if bridge br1 already exists...
bridge name	bridge id		STP enabled	interfaces
br1		can't get info No such device

```